internal/storage/storageutil/custom_retry.go

```diff
@@ -16,12 +16,14 @@ package storageutil
 
 import (
 	"cloud.google.com/go/storage"
+	"github.com/googlecloudplatform/gcsfuse/v2/internal/logger"
 	"google.golang.org/api/googleapi"
 )
 
 func ShouldRetry(err error) (b bool) {
 	b = storage.ShouldRetry(err)
 	if b {
+		logger.Info("Retrying for the error: %w", err)
```

Comment:
should be logger.Infof ? If you see the sample logs in description, "%w" is coming out separate from error.

Reply:
%w is not working, so changed it to %v.